overlord/devicestate/devicemgr.go

```diff
@@ -1772,6 +1793,7 @@ var seedFailureFmt = `seeding failed with: %v. This indicates an error in your d
 
 // Ensure implements StateManager.Ensure.
 func (m *DeviceManager) Ensure() error {
+	logger.Trace("Ensure", "manager", "devicemgr", "func", "Ensure")
```

Comment:
do we need the "func" here?  if it's left out we can assume it's Ensure no given the the kind